sycl/include/CL/sycl/detail/pi.h

```diff
@@ -855,6 +855,17 @@ pi_result piEnqueueMemUnmap(
   const pi_event * event_wait_list,
   pi_event *       event);
 
+pi_result piGetDeviceFunctionPointer(
```

Comment:
Please document this new PI interface with doxygen (I know there is no good example in PI to follow, but let's start adding new interfaces with the documentation from the beginning.

Reply:
Done